pylib/anki/importing/anki2.py

```diff
@@ -344,7 +344,7 @@ def _importCards(self) -> None:
             card[4] = intTime()
             card[5] = usn
             # review cards have a due date relative to collection
-            if card[7] in (2, 3) or card[6] == 2:
+            if card[7] in (2, 3) or card[6] == CARD_TYPE_LRN:
```

Comment:
This is wrong

Reply:
I corrected the entire line. I don't know how I got this one so wrong